index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tochukwu Orizu</title>
+    <link rel="stylesheet" href="/styles/style.css">
+
+</head>
+<body>
+    <header class="header">
+        <div class="logo">My logo</div>
+        <nav class="navigation">
+            <img src="/img/Icons/Union.png" alt="#">
+        </nav>
+    </header>
+
+    <section class="container">
+        <h1 class="box1">I'm Tochukwu Glad to see you!</h1>
+        <p class="box2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="box3">LET'S CONNECT</p>
+    </section>
+    <footer class="socials">
+        <img src="/img/socialmediaicons.png" alt="#"> 
+    </footer>
+</body>
+</html>
```

Comment:
- [ ] Great job removing the unncessary code, however, your padding and margin is still looking fairly different from the original design:

Template: 
![Captura de pantalla 2022-03-18 a la(s) 9 43 21 p  m](https://user-images.githubusercontent.com/83717285/159103834-8ffc631c-c8e2-4307-a407-c10ead67898f.png)

Your design:
![Captura de pantalla 2022-03-18 a la(s) 9 43 39 p  m](https://user-images.githubusercontent.com/83717285/159103848-df265806-70f1-46a9-96e4-fb5f8d95efc8.png)

Please address this problem and you will be done! 💯